client-sdk/ts-web/core/playground/src/startPlayground.mjs

```diff
@@ -1,10 +1,8 @@
 // @ts-check
 
-import * as oasis from './../..';
-
-const nic = new oasis.client.NodeInternal('http://localhost:42280');
-
-export const playground = (async function () {
+/** @param {import('./../..')} oasis */
+export async function startPlayground(oasis) {
```

Comment:
we were interested in fixing the import so that we don't need to turn this into a function

Reply:
oh or at least removing the parameter. dunno if you prefer it to be a function.

Reply:
Parameter: it was needed because nodejs and cypress having incompatible import syntaxes. But since this is an `npm workspace`, replacing "./../.." with "@oasisprotocol/client" should work and be compatible.

I'll remove the parameter, but I do prefer it as a function

Reply:
thanks. yeah, we avoid side effects everywhere else too, so it's consistent to move it into a function